tests/components/zha/test_registries.py

```diff
@@ -113,10 +125,9 @@ def channels(channel):
         ),
     ],
 )
-def test_registry_matching(rule, matched, zha_device, channels):
+def test_registry_matching(rule, matched, channels):
     """Test strict rule matching."""
-    reg = registries.ZHAEntityRegistry()
-    assert reg._strict_matched(zha_device, channels, rule) is matched
+    assert rule.strict_matched(MANUFACTURER, MODEL, channels) is matched
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5051600)